templates_jinja2/team_details.html

```diff
@@ -142,6 +142,8 @@ <h3><a href="/event/{{ comp.event.key_name }}">{{ comp.event.name }}</a></h3>
                   </ul>
                   {% endif %}
                 </p>
+                <a class="btn btn-default" href="https://www.youtube.com/watch_videos?video_ids={{comp.video_ids}}" target="_blank">
```

Comment:
Only show this if there's at least one video published. Otherwise you're gonna run into a nasty `400` error.